etc/9.yaml

```diff
@@ -0,0 +1,17 @@
+---
+0:
```

Comment:
instead of using 0 here, you can just use the `name` as the key to the yaml dict

Reply:
same for above

Reply:
maybe actually just have a setups folder than has each setup as its own file in this similar pattern?
and then a master file that contains player size to setup name that can be loaded and kept in memory